mlir/lib/Dialect/Rock/Transforms/AccelEmitter.cpp

```diff
@@ -600,6 +600,275 @@ Value MfmaEmitter::wrapLDSBufferForLoad(OpBuilder &b, Location loc,
   return transform(b, buffer, ldsRead);
 }
 
+RegsAsMatrixSubTiles MfmaEmitter::createAccelGemmOperandTransforms(
```

Comment:
Should this be used in the GridwiseGemmV2 lowering as well?

Reply:
Then I'm not understanding what this function does.

I thought it was the (tid, iter) -> (i, j) transform

Reply:
We've got a lot of these utilities that the attention pass uses that could be used in the general gemm pass, and I know there's already a general ticket for that refactoring.

Reply:
Yep that is right.

Well at least this one is in a common place (AccelEmitter.cpp), so what is remaining is to do the code to use them in gemm lowering. 

Im a bit purposefully avoiding/delaying that work because Im constantly changing these utils until I have perf on attention kernels. Therefore I think it makes sense to do the refactor last when we have the attention kernels more or less ready -- so that we dont have to carry the burden of changing everything together, if that makes sense.

Reply:
Yeah, entirely fair.

Which probably explains why the transform slicer hasn't happened yet.

Reply:
that and I think @ravil-mobile is busy with split-k stuff. https://github.com/ROCm/rocMLIR-internal/issues/1303